content/en/docs/concepts/configuration/scheduler-perf-tuning.md

```diff
@@ -0,0 +1,113 @@
+---
+reviewers:
+- bsalamat
+title: Scheduler Performance Tuning
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="1.12" >}}
+
+Kube-scheduler is the Kubernetes default scheduler. It is responsible for
```

Comment:
Do we need this statement?

Reply:
It is in the overview section. It would be helpful for someone unfamiliar with the topic to know what they are reading about.